src/TinyIoC/TinyIoC.cs

```diff
@@ -3907,8 +3924,13 @@ private static bool IsValidAssignment(Type registerType, Type registerImplementa
             {
                 if (registerType.IsInterface())
                 {
+#if PORTABLE
+                    if (!registerImplementation.GetInterfaces().Any(t => t.Name == registerType.Name))
+                        return false;
+#else
                     if (!registerImplementation.FindInterfaces((t, o) => t.Name == registerType.Name, null).Any())
                         return false;
+#endif

```

Comment:
I copied this from the previous version as this was changed since april and FindInterfaces does exist in the pcl profile